citesphere/src/main/java/edu/asu/diging/citesphere/core/service/impl/CitationManager.java

```diff
@@ -430,6 +430,33 @@ public CitationResults getGroupItems(IUser user, String groupId, String collecti
 
     }
 
+    /**
+     * Get all citations that will be matched to citation author uri
+     * , editor uri or contributor uri with that of the query parameter uri.
+     *
+     * @param user    User who is accessing Zotero.
+     * @param uri     Uri is the query parameter.
```

Comment:
what query parameter? at this point, it doesn't matter where the URI is coming from (same above in the description).